Tests/ReactiveSwiftTests/SignalProducerLiftingSpec.swift

```diff
@@ -1749,19 +1727,132 @@ class SignalProducerLiftingSpec: QuickSpec {
 						case .failed:
 							break
 						}
-					}
+				}
```

Comment:
I actually meant that the bracket should aligned with `.start`. Sorry if I wasn't clear enough. 😸